src/main/resources/application.yml

```diff
@@ -6,10 +6,12 @@ integration:
     connectTimeout: 10
     readTimeout: 20
 logbook:
-  exclusionfilters:
+  body-filters:
     json-path:
-      '$..file': '[base64]'
-      '$..content': '[base64]'
+     - key: '$..file'
```

Comment:
`$..[content,file]` ??

Reply:
Not able to reproduce the behaviour of this.
closest I get to compile and actually filter things is  `"['file','content']"` which seems to be good enough for this service since fields don't seem to be nested.